### ISSUE TITLE
build_ios_app fails after update xCode from 11.7 to 12.0

### ISSUE THREAD

**Issue by ryabchikova** [fastlane/fastlane#17296]
Before update xcode to 12.0 my fastlane script for Firebase distribution works without errors.
Now I have error for build_ios_app action:

bundler: failed to load command: fastlane (/usr/local/bin/fastlane)
ArgumentError: [!] same file: /var/folders/_3/bvmbdcnd5q5fpxdmzf50d69c0000gq/T/gym_output20200922-26715-1nxriiz/Касса.ipa and /var/folders/_3/bvmbdcnd5q5fpxdmzf50d69c0000gq/T/gym_output20200922-26715-1nxriiz/Касса.ipa

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 10.15.6                                                        |
| Ruby                        | 2.6.3                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.24.3 (Apple Git-128)                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | Mac OS X 10.15.6 (19G2021)                                     |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib |
| OpenSSL Version             | LibreSSL 2.8.3                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 12.0                                                           |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./Fastfile`</summary>

```ruby
# Uncomment the line if you want fastlane to automatically update itself
# update_fastlane

default_platform(:ios)

platform :ios do

  def install_pods
    cocoapods(
      clean_install: true,
      podfile: "Podfile",
      try_repo_update_on_error: true,
      silent: true
    )
  end

  def build_notes()
    commit = last_git_commit
    notes = "commit: #{commit[:abbreviated_commit_hash]}\n"
    notes.concat("author: #{commit[:author]}\n")
    notes.concat("#{commit[:message]}")
    puts notes
    return notes
  end

  lane :lint do
    swiftlint(
      reporter: "html",
      strict: true,
      quiet: true,
      output_file: "fastlane/Report/swiftlint.html"
    )
  end

  lane :firebase do |options|
    cert
    sigh(
      adhoc: true,
      force: true
    )
    increment_build_number(build_number: options[:build_number])
    install_pods()
    build_ios_app(
        scheme: "Kassa2.0",
        clean: true,
        silent: true,
        suppress_xcode_output: true
    )
    firebase_app_distribution(
      app: "",
      groups: "ios_development",
      release_notes: build_notes(),
      debug: false
    )
    clean_build_artifacts
  end

  lane :my_dbg do
  end

end

```
</details>

<details><summary>`./Appfile`</summary>

```ruby
app_identifier("...")
apple_id("...")
team_id("...")

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.160.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                                    | Version | Update-Status |
| ----------------------------------------- | ------- | ------------- |
| fastlane-plugin-firebase_app_distribution | 0.2.1   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                                       | Version      |
| ----------------------------------------- | ------------ |
| did_you_mean                              | 1.3.0        |
| slack-notifier                            | 2.3.2        |
| rouge                                     | 2.0.7        |
| xcpretty                                  | 0.3.0        |
| terminal-notifier                         | 2.0.0        |
| plist                                     | 3.5.0        |
| addressable                               | 2.7.0        |
| multipart-post                            | 2.0.0        |
| word_wrap                                 | 1.0.0        |
| tty-cursor                                | 0.7.1        |
| tty-spinner                               | 0.9.3        |
| babosa                                    | 1.0.3        |
| colored                                   | 1.2          |
| highline                                  | 1.7.10       |
| commander-fastlane                        | 4.4.6        |
| faraday                                   | 1.0.1        |
| faraday_middleware                        | 1.0.0        |
| gh_inspector                              | 1.1.3        |
| mini_magick                               | 4.10.1       |
| rubyzip                                   | 2.3.0        |
| security                                  | 0.1.3        |
| xcpretty-travis-formatter                 | 1.0.0        |
| bundler                                   | 2.1.4        |
| naturally                                 | 2.2.0        |
| simctl                                    | 1.6.8        |
| uber                                      | 0.1.0        |
| declarative                               | 0.0.20       |
| declarative-option                        | 0.1.0        |
| representable                             | 3.0.4        |
| retriable                                 | 3.1.2        |
| mini_mime                                 | 1.0.2        |
| signet                                    | 0.14.0       |
| httpclient                                | 2.8.3        |
| google-api-client                         | 0.38.0       |
| nanaimo                                   | 0.3.0        |
| colored2                                  | 3.1.2        |
| claide                                    | 1.0.3        |
| CFPropertyList                            | 3.0.2        |
| atomos                                    | 0.1.3        |
| xcodeproj                                 | 1.18.0       |
| unicode-display_width                     | 1.7.0        |
| terminal-table                            | 1.8.0        |
| public_suffix                             | 4.0.6        |
| tty-screen                                | 0.8.1        |
| excon                                     | 0.76.0       |
| unf_ext                                   | 0.0.7.7      |
| unf                                       | 0.1.4        |
| domain_name                               | 0.5.20190701 |
| http-cookie                               | 1.0.3        |
| faraday-cookie_jar                        | 0.0.7        |
| fastimage                                 | 2.2.0        |
| json                                      | 2.3.1        |
| dotenv                                    | 2.7.6        |
| jwt                                       | 2.2.2        |
| multi_json                                | 1.15.0       |
| os                                        | 1.1.1        |
| memoist                                   | 0.16.2       |
| googleauth                                | 0.13.1       |
| rake                                      | 13.0.1       |
| digest-crc                                | 0.6.1        |
| google-cloud-errors                       | 1.0.1        |
| google-cloud-env                          | 1.3.3        |
| google-cloud-core                         | 1.5.0        |
| google-cloud-storage                      | 1.28.0       |
| emoji_regex                               | 3.0.0        |
| aws-eventstream                           | 1.1.0        |
| aws-sigv4                                 | 1.2.2        |
| aws-partitions                            | 1.374.0      |
| jmespath                                  | 1.4.0        |
| aws-sdk-core                              | 3.107.0      |
| aws-sdk-kms                               | 1.38.0       |
| aws-sdk-s3                                | 1.81.0       |
| forwardable                               | 1.2.0        |
| logger                                    | 1.3.0        |
| stringio                                  | 0.0.2        |
| ipaddr                                    | 1.2.2        |
| openssl                                   | 2.1.2        |
| ostruct                                   | 0.1.0        |
| strscan                                   | 1.0.0        |
| date                                      | 2.0.0        |
| fileutils                                 | 1.1.0        |
| etc                                       | 1.0.1        |
| io-console                                | 0.4.7        |
| zlib                                      | 1.0.0        |
| libxml-ruby                               | 3.1.0        |
| rexml                                     | 3.1.9        |
| psych                                     | 3.1.0        |
| mutex_m                                   | 0.1.0        |
| webrick                                   | 1.4.2        |
| fastlane-plugin-firebase_app_distribution | 0.2.1        |
</details>

*generated on:* **2020-09-22**
</details>

**Comment by Strate** [comment]
@ryabchikova hello, did you manage somehow this issue?

**Comment by ryabchikova** [comment]
> @ryabchikova hello, did you manage somehow this issue?

No, it doesn't work yet. We are back to xCode 11.7 in our CI environment.

**Comment by Strate** [comment]
#17398

**Comment by ryabchikova** [comment]
I tried to test with 
gem 'fastlane', :git => 'https://github.com/mollyIV/fastlane.git', :branch => 'gym-fix-building-projects-with-unicode-characters-in-name'

But met another problem, like this https://github.com/CocoaPods/CocoaPods/issues/9884
So, I'll waiting for all the fixes and releases.

One one clarification: out project has Display name and Product name on Russian ("Касса"), but project name on English ("Kassa2.0"). May be we just should change  Product name, I'll check it later.

**Comment by ryabchikova** [comment]
My problem fixed update fastlane to 2.163.0, thanks!

**Comment by rogerluan** [comment]
Thanks for confirming @ryabchikova 🤗
